src/main/java/org/cryptomator/cryptofs/CryptoFileSystemProperties.java

```diff
@@ -80,6 +81,15 @@ public class CryptoFileSystemProperties extends AbstractMap<String, Object> {
 
 	static final String DEFAULT_MASTERKEY_FILENAME = "masterkey.cryptomator";
 
+	/**
+	 * Key identifying the function to call for notifications.
+	 *
+	 * @since 2.9.0
+	 */
+	public static final String PROPERTY_EVENT_CONSUMER = "notificationConsumer";

```

Comment:
Change the value as well, since being used as a map key it is essentially part of the public API

```suggestion
	public static final String PROPERTY_EVENT_CONSUMER = "fsEventConsumer";
```